Make test fails on master branch a4c29ea7efd292a5105f4a681eadfa0a36204a55

Make test fails on master branch a4c29ea7efd292a5105f4a681eadfa0a36204a55.

alpha-wallet-ios/AlphaWalletTests/Transfer/ViewModels/RequestViewModelTests.swift:11:58: error build: Missing argument for parameter 'domainResolutionService' in call
```swift
let viewModel = RequestViewModel(account: account)
```
RequestViewModelTests.swift:11

--- a/AlphaWalletTests/Transfer/ViewModels/RequestViewModelTests.swift
+++ b/AlphaWalletTests/Transfer/ViewModels/RequestViewModelTests.swift
@@ -3,13 +3,26 @@
 import XCTest
 @testable import AlphaWallet
 import AlphaWalletFoundation
+import Combine
 
 class RequestViewModelTests: XCTestCase {
+    private var cancelable = Set<AnyCancellable>()
 
     func testMyAddressText() {
         let account: Wallet = .make()
-        let viewModel = RequestViewModel(account: account)
+        let viewModel = RequestViewModel(account: account, domainResolutionService: FakeDomainResolutionService())
+        let expectation = self.expectation(description: "View model state has resolved")
+        let output = viewModel.transform(input: .init(copyEns: .empty(), copyAddress: .empty()))
 
-        XCTAssertEqual(account.address.eip55String, viewModel.myAddressText)
+        output.viewState
+            .sink { viewState in
+                XCTAssertEqual(account.address.eip55String, viewState.address, "matches address value")
+                //XCTAssertNotNil(viewState.qrCode, "has qr code")
+                if viewState.qrCode != nil {
+                    expectation.fulfill()
+                }
+            }.store(in: &cancelable)
+
+        wait(for: [expectation], timeout: 20)
     }
 }

